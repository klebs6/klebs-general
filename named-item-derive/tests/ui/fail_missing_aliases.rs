// ---------------- [ File: tests/ui/fail_missing_aliases.rs ]
// Should fail because `aliases="true"` but no aliases: Vec<String> field

use named_item_derive::NamedItem;

#[derive(NamedItem)]
#[named_item(aliases="true")]
struct MissingAliasesField {
    name: String,
}

fn main() {}
