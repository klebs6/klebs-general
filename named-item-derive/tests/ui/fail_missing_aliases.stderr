error: aliases=true but no `aliases: Vec<String>` field found.
 --> tests/ui/fail_missing_aliases.rs:8:8
  |
8 | struct MissingAliasesField {
  |        ^^^^^^^^^^^^^^^^^^^
